ui/src/app/utils/resourceActions.ts

```diff
@@ -0,0 +1,48 @@
+export class NotebookActions {
```

Comment:
nit: you shouldn't need the class wrapper here,  you could just export the lists.